content/docs/started/getting-started-gke.md

```diff
@@ -1,378 +0,0 @@
-+++
-title = "Kubernetes Engine for Kubeflow"
```

Comment:
We need to add redirects to catch incoming links from external docs. We also need to fix all links within our own docs that point to this deleted page.

Reply:
Can you provide some direction or example an how I do this?

Reply:
Sure! Define the redirects in this file, following the pattern of the other entries:
https://github.com/kubeflow/website/blob/master/content/_redirects

This doc has guidelines on the processing order of redirects (basically, they're processed from the top down, and you should put more specific redirects before those that use wild cards etc):
https://www.netlify.com/docs/redirects/

Reply:
I think this is no longer needed because this page now still exists its just pointing to the other docs.